File: files/200-299/252/CH-252.R
library(tidyverse)
library(readxl)

path = "files/200-299/252/CH-252 Switching Value.xlsx"
input = read_excel(path, range = "C2:D9")
test  = read_excel(path, range = "F2:G9")

result = input %>%
  mutate(Importance = recode(Importance,
                             "Very low" = 1,
                             "Low" = 2,
                             "Moderate" = 3,
                             "High" = 4,
                             "Very high" = 5))

all.equal(result, test)
#> [1] TRUE

File: files/200-299/252/CH-252.py
import pandas as pd

path = "200-299/252/CH-252 Switching Value.xlsx"
input = pd.read_excel(path, usecols="C:D", skiprows=1, nrows=8)
test = pd.read_excel(path, usecols="F:G", skiprows=1, nrows=8).rename(columns=lambda x: x.replace('.1', ''))

result = input.copy()
result['Importance'] = result['Importance'].str.strip().replace({
    "Very low": 1, "Low": 2, "Moderate": 3, "High": 4, "Very high": 5
})

print(result.equals(test)) # True